tasks/configure.yml

```diff
@@ -138,6 +138,48 @@
   with_items: "{{ haproxy_userlists }}"
   when: haproxy_userlists is defined
 
+## CREATE ERROR PAGES
+
+- name: Add error directory
+  file:
+    path: "{{ haproxy_config_dir }}/errors"
+    state: directory
+
+- name: '400'
```

Comment:
The name is too short to me. My advice is to prefix with the tasksbook name `configure`, and to start with _Ensure ..._.

My suggestion is : `- name: Configure - Ensure http 400 error file is present.`

Reply:
The taskbook as no prefix right now so I am leaving this part for another PR.

Reply:
I renamed the tasks.